apps/src/blockly/addons/cdoBlockDragger.js

```diff
@@ -50,6 +51,8 @@ export default class BlockDragger extends GoogleBlockly.BlockDragger {
     super.updateCursorDuringBlockDrag_();
     if (this.draggedConnectionManager_.wouldDeleteBlock()) {
       this.workspace_.trashcan.setLidOpen(true);
+    } else {
+      this.workspace_.trashcan.setLidOpen(false);
     }
```

Comment:
nit: this could be simplified to 

```javascript
const wouldDelete = this.draggedConnectionManager_.wouldDeleteBlock();
this.workspace_.trashcan.setLidOpen(wouldDelete);
```